### ISSUE TITLE
Broken tag search feature

### ISSUE THREAD

**Issue by kokanek** [thepracticaldev/dev.to#4197]
The search feature for tags while writing an article and tagging it looks broken when the full keyword intended to be used is typed in.

Whenever I want to tag my article with a particular tag and I start typing the tag, it appears in the suggested tags in the dropdown but once I type the full tag name, it seems to dissappear from the suggested tags dropdown. 

**To Reproduce**

Steps to reproduce

1. Click on 'Write a Post' in the upper right hand corner to start writing a post.
2. In the text box that says "4 tags max... " start typing tags.
3. Type "reac" and we can see "react" in the suggestions.
4. Type "react" and the "react" tag disappears from the suggestions.

**Expected behavior**

The expected behavior is that the tag "react" should still be visible in the dropdown of tags even after "react" is typed in the search box.

**Screenshots**

![chrome_A6B8PKC3nt](https://user-images.githubusercontent.com/16059151/66070468-020c4580-e56f-11e9-98e4-d758d6d8ee1e.png)
![chrome_TTF8Sn7E18](https://user-images.githubusercontent.com/16059151/66070470-020c4580-e56f-11e9-9a8d-34f2878c90df.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Chrome 
- Version:

**Comment by torianne02** [comment]
I created a solution to this issue. While attempting to git add and commit to my local repo and branch I came across failing eslint tests. It looks like a lot of maintenance work. I was wondering if I needed to fix all of these errors, or is there a way to bypass these tests? I have included a screenshot of the tests failing (couldn't fit them 

<img width="722" alt="Screen Shot 2019-10-03 at 12 07 36 PM" src="https://user-images.githubusercontent.com/37308853/66156405-729a7d00-e5d6-11e9-987f-aeb5f471a95b.png">
